app/src/main/kotlin/com/simplemobiletools/dialer/dialogs/FilterContactSourcesDialog.kt

```diff
@@ -31,6 +33,9 @@ class FilterContactSourcesDialog(val activity: SimpleActivity, private val callb
 
         contactHelper.getContacts(getAll = true) { contacts ->
```

Comment:
i guess we shouldnt use "contacts" here again, it can lead to bugs as there is already a more global "contacts"